src/wp-admin/install.php

```diff
@@ -299,119 +147,286 @@ function display_setup_form( $error = null ) {
 			break;
 		}
 
-		// Deliberately fall through if we can't reach the translations API.
-
-	case 1: // Step 1, direct link or from language chooser.
+	// Display the final setup screen (site title, admin user).
+	case 1:
+		// If language is present, ensure it's loaded.
 		if ( ! empty( $language ) ) {
 			$loaded_language = wp_download_language_pack( $language );
 			if ( $loaded_language ) {
 				load_default_textdomain( $loaded_language );
 				$GLOBALS['wp_locale'] = new WP_Locale();
 			}
 		}
-
+		// Add another script... Is this used?
 		$scripts_to_print[] = 'user-profile';
-
+		// Display the page header.
 		display_header();
-?>
-<h1><?php _ex( 'Welcome', 'Hello' ); ?></h1>
-<p><?php _e( 'Welcome to the ClassicPress installation process! ClassicPress brings a rich set of features expected from an advanced modern CMS while keeping the old and proven way to compose and manage your content. Fill in the information below and the impressive power of the ClassicPress publishing platform will be ready to serve your business.' ); ?></p>
-
-<h2><?php _e( 'Information needed' ); ?></h2>
-<p><?php _e( 'Please provide the following information. Don&#8217;t worry, you can always change these settings later.' ); ?></p>
-
-<?php
+		// Add a main title.
+		echo '<h1>'. __( 'Admin Setup') .'</h1>';
+		// Add descriptive text.
+		echo '<p>' . __( 'Give your site a great title and fill out your administrator account details. Take note of your username and password – you will need these again in a moment.' ) . '</p>';
+		// Print the setup form.
 		display_setup_form();
+		// I rest my case.
 		break;
+
+	// Process the final setup step and redirect.
 	case 2:
+		// Ensure a language is loaded.
 		if ( ! empty( $language ) && load_default_textdomain( $language ) ) {
 			$loaded_language = $language;
 			$GLOBALS['wp_locale'] = new WP_Locale();
 		} else {
 			$loaded_language = 'en_US';
 		}
-
-		if ( ! empty( $wpdb->error ) )
-			wp_die( $wpdb->error->get_error_message() );
-
+		// Was there a database error? If so, bail with error message.
+		if ( ! empty( $wpdb->error ) ) {
+			display_header( 'cp-installation-error' );
+			echo '<h1>'. __('Configuration Error') . '</h1>';
+			echo '<p>' . $wpdb->error->get_error_message() . '</p>';
+			display_footer();
+		}
+		// Add another script to the mix... Is this used?
 		$scripts_to_print[] = 'user-profile';
-
+		// Print the branded page header.
 		display_header();
-		// Fill in the data we gathered
+		// Get and pare submitted data.
```

Comment:
`parse` instead of `pare`?

Reply:
Pare...to trim. I've reworded it for clarity.